docs/GettingStarted.md

```diff
@@ -300,8 +298,36 @@ You can also get the root span of the current active trace using the `active_roo
 current_root_span = Datadog.tracer.active_root_span
 current_root_span.set_tag('my_tag', 'my_value') unless current_root_span.nil?
 ```
+## Compatibility
```

Comment:
This `Compatibility` section looks redundant; result of a merge conflict? See top of document.

Reply:
this is actually to make it show up as an h2 in our docs. we do this often, but will this work in the ruby docs?

Reply:
I don't think `##` is an issue at all, but this information appears twice in the document? On line 73 and 301, with conflicting information. Ideally this should only appear once, near the top of the document (so users don't have to scan far to know whether they can use the library or not.)

Reply:
oh! I see what you mean now. removed the duplicate content!